src/chigame/games/models.py

```diff
@@ -471,3 +471,21 @@ def save(self, *args, **kwargs):
 
     def __str__(self):
         return "Message from " + str(self.sender) + ": " + self.content
+
+
+class Review(models.Model):
+    "Represents a game review"
+    title = models.TextField(blank=True, null=True)
+    review = models.TextField(blank=True, null=True)
+    rating = models.PositiveSmallIntegerField(blank=True, null=True)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    game = models.ForeignKey(Game, on_delete=models.CASCADE)
+    is_public = models.BooleanField(default=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
+        return f"Review {self.id} by {self.user} for {self.game}: {self.review}"
+
+    def clean(self):
```

Comment:
In order for the the custom `clean` method to take effect, you also need to write a custom `save` method. See either the `Game` or `Lobby` model as an example of how to do this.

Reply:
I've made this adjustment. Thanks for looking out!

Reply:
When creating reviews via the admin page I can leave the title, summary, and rating blank. Were you able to test this validation and get it to work?